moonscript/base.moon

```diff
@@ -34,6 +34,25 @@ to_lua = (text, options={}) ->
 
   code, ltable
 
+loadstring = (...) ->
+  options, str, chunk_name, mode, env = get_options ...
+  chunk_name or= "=(moonscript.loadstring)"
+
+  if string.sub(chunk_name,-8)==".litmoon"
```

Comment:
Why did you decide to use chunkname to determine the format? Seems strange. What about just having a separate `loadstring` function for litmoon source?